### PR TITLE
[New] Add feather-icons@3.2.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by venkatramanareddy**
Pull request for issue: #11753

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/colebemis/feather
 * Official website (optional, not the repository): https://feathericons.com/
 * NPM package url (optional): [https://www.npmjs.com/package/feather](https://www.npmjs.com/package/feather)
 * License and its reference:  [MIT](https://github.com/colebemis/feather/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 192
   - Count of stars: 9674
   - Count of forks: 267
 * NPM download stats (optional):
   - Downloads in the last day: 191
   - Downloads in the last week: 833
   - Downloads in the last month: 2144

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [X] More than 800 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by venkatramanareddy** [comment]
#11753 Added feather.js and feather.min.js

**Comment by extend1994** [comment]
Hi @venkatramanareddy May I know how did you commit the files? via GitHub GUI or local git commands?

**Comment by venkatramanareddy** [comment]
This was my first contribution for cdnjs. I've done them using git commands.

File: ajax/libs/feather-icons/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "feather-icons",
+  "filename": "feather.min.js",
+  "keywords": [
+    "popular",
+    "icons"
+  ],
+  "version": "3.3.2",
```
Comment (extend1994) [inline ajax/libs/feather-icons/package.json]: The latest version is `3.2.2` :) 
So the version folder needs to be revised too. You can directly remove the folder and grab the files using auto-update config like step 2 and 3. 💪

**Comment by extend1994** [comment]
@venkatramanareddy The author replied! All we need is `feather`. Here are the steps you can follow if work locally:
1. `git reset --hard HEAD^` reset the state to https://github.com/cdnjs/cdnjs/pull/11764/commits/a0aa97618d51a1a63da5dc93aa1c983292b3af07
2.  `git rebase -i HEAD~2` and choose `fixup` to the commit https://github.com/cdnjs/cdnjs/pull/11764/commits/a0aa97618d51a1a63da5dc93aa1c983292b3af07
3. Complete the following items: 
- remove `3.3.2/feather.js` and `3.3.2/feather.min.js` in first commit.
- enrich package.json as the above requests regarding `package.json`
- add `dist/feather.js`, `dist/feather.min.js`, `dist/feather.js.map` and `dist/feather.min.js.map` from feather-icons@3.2.2 on npm
- `git add .`
- `git commit --amend` and use the commit message
```
Add feather-icons@3.3.2 w/ npm auto-update

close #11753, cc @colebemis
```
Thank you.
If you commit the file via GitHub GUI instead of local git commands, I can help you do the above. 😄

File: ajax/libs/feather-icons/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "feather-icons",
+  "filename": "feather.min.js",
+  "keywords": [
+    "popular",
+    "icons"
```
[the remark this answers is not in the document]
Comment (extend1994) [reply ajax/libs/feather-icons/package.json]: Let's do this change.

File: ajax/libs/feather-icons/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  ],
+  "version": "3.3.2",
+  "description": "Simply beautiful open source icons",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/colebemis/feather.git"
+  },
+  "author": "Cole Bemis <cole@colebemis.com> (http://colebemis.com)",
+  "license": "MIT"
```
[the remark this answers is not in the document]
Comment (extend1994) [reply ajax/libs/feather-icons/package.json]: Let's do this change.

[the remark this answers is not in the document]
Comment (extend1994) [reply ajax/libs/feather-icons/package.json]: @venkatramanareddy Let's do this change.

File: ajax/libs/feather-icons/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (extend1994) [reply ajax/libs/feather-icons/package.json]: @venkatramanareddy Let's do this change.

**Comment by extend1994** [comment]
@venkatramanareddy I noticed that you might changed you username recently? If you mind the contribution to CDNJS. We might need to do something to deal with it.

File: ajax/libs/feather-icons/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (sufuf3) [reply ajax/libs/feather-icons/package.json]: @venkatramanareddy Let's add this part.

**Comment by colebemis** [comment]
Hi, all 👋 I’m the creator of Feather. I was just curious about the state of this pull. Is it still in development?

**Comment by extend1994** [comment]
Hi @colebemis thanks for attention here 😄 Yes it is still in process. We are waiting for @venkatramanareddy to finish this. 
Since this pull request is for an high priority issue. @venkatramanareddy I will help fix this pull request if you don't have time or we don't have your response in at most 7 days.

**Comment by venkatramanareddy** [comment]
Sorry that I didn't find much time to correct. Are there any changes required?

**Comment by extend1994** [comment]
@maruilian11 Please see https://github.com/colebemis/feather/issues/178. We only need js files. Help revise the PR title. It was my carelessness.

**Comment by maruilian11** [comment]
@extend1994 @venkatramanareddy If we don't need `icon`, then we should avoid adding it in `filemap`.

**Comment by venkatramanareddy** [comment]
I ran the auto-update.js for feather-icons with ```files``` as ```**/**``` and following files got listed-
1. feather.js
2. feather.js.map
3. feather.min.js
4. feather.min.js.map

Should we be adding the filters for ```icon```? If then how

**Comment by maruilian11** [comment]
@venkatramanareddy Try remove `3.2.2` folder and run `auto-update.js` again. Do you still get this?
```
├── 3.2.2
│   ├── feather.js
│   ├── feather.js.map
│   ├── feather.min.js
│   └── feather.min.js.map
```
It's mine:
```
├── 3.2.2
│   ├── feather.js
│   ├── feather.js.map
│   ├── feather.min.js
│   ├── feather.min.js.map
│   └── icons
│       └── feather.svg
```
You can use thie filemap to avoid adding `icons`:
`"**/feather*.+(js|css|map)"`

**Comment by PeterDaveHello** [comment]
@maruilian11 try to **exclude** file types we don't need in the `dist` path, not to **include** the files we need.

**Pull request merged**